packages/dd-trace/src/config.js

```diff
@@ -239,7 +239,8 @@ class Config {
     options = {
       ...options,
       appsec: options.appsec != null ? options.appsec : options.experimental?.appsec,
-      iast: options.iast != null ? options.iast : options.experimental?.iast
+      iast: options.iast != null ? options.iast : options.experimental?.iast,
+      llmobs: options.llmobs != null ? options.llmobs : options.experimental?.llmobs
```

Comment:
we're not sure if we want to label this as experimental yet. once i have a PR up for the type definitions, we'll know more, and can maybe change this, but for now we'll check if the LLMObs config is on the main config or `experimental`